Pipfile

```diff
@@ -7,6 +7,7 @@ name = "pypi"
 django = "*"
 rdflib = "*"
 requests = "*"
+qwikidata = "*"
```

Comment:
Why do we need `qwikidata`? What does it do that `rdflib` doesn't do?

~~Moreover, what does it do that the `query.wikidata.org` endpoint doesn't?~~

Literally all it does (or rather, all you're using it for) is make a GET request to the endpoint, just like we're doing in the `test_wikidata.py` command. You can see the source code for the qwikidata `return_sparql_query_results` function that you use [here](https://github.com/kensho-technologies/qwikidata/blob/develop/qwikidata/sparql.py#L10). As such, I would remove this dependency and instead just write our own function that makes a GET request to the endpoint and returns the result.